### ISSUE TITLE
Exclude non-public asset files from release bundling

### ISSUE THREAD

**Issue by edurne85** [edurne85/wowfinder#298]
When bundling a release, all contents within the local `assets` folder are being included, even if they are not part of the repository. The bundles should exclude non public files, specifically:

* `assets/Characters/*`
* `assets/Adventures/*`
* `assets/Items/gear/unique/**` - Only once the export functionality is updated to include this data
